nipype/interfaces/mrtrix3/preprocess.py

```diff
@@ -241,6 +242,151 @@ def _format_arg(self, name, trait_spec, value):
                 return f"-{trait_spec.argstr}"
         return super()._format_arg(name, trait_spec, value)
 
+
+class DWIPreprocInputSpec(MRTrix3BaseInputSpec):
+    in_file = File(
+        exists=True, argstr="%s", position=-10, mandatory=True, desc="input DWI image"
+    )
+    out_file = File(
+        "preproc.mif",
+        argstr="%s",
+        mandatory=True,
+        position=-9,
+        usedefault=True,
+        desc="output file after preprocessing",
+    )
+    rpe_options = traits.Enum(
+        "-rpe_none",
+        "-rpe_pair",
+        "-rpe_all",
+        "-rpe_header",
+        argstr="%s",
+        position=-8,
+        mandatory=True,
+        desc="Specify acquisition phase-encoding design, one of the -rpe_* options must be provided",
+    )
+    pe_dir = traits.Str(
+        argstr="-pe_dir %s",
+        position=-7,
+        mandatory=True,
+        desc="Specify the phase encoding direction of the input series, can be a signed axis number (e.g. -0, 1, +2), an axis designator (e.g. RL, PA, IS), or NIfTI axis codes (e.g. i-, j, k)",
+    )
+    ro_time = traits.Float(
+        argstr="-readout_time %f",
+        position=-6,
+        desc="Total readout time of input series (in seconds)",
+    )
+    in_epi = File(
+        exists=True,
+        argstr="-se_epi %s",
+        position=-5,
+        desc="Provide an additional image series consisting of spin-echo EPI images, which is to be used exclusively by topup for estimating the inhomogeneity field (i.e. it will not form part of the output image series)",
+    )
+    align_seepi = traits.Bool(
+        argstr="-align_seepi",
+        position=-4,
+        desc="Achieve alignment between the SE-EPI images used for inhomogeneity field estimation, and the DWIs",
+    )
+    eddy_options = traits.Str(
+        argstr="-eddy_options %s",
```

Comment:
Looks from the description these are supposed to be quoted?

```suggestion
        argstr="-eddy_options \"%s\"",
```